src/plugins/xfconf/README.md

```diff
@@ -0,0 +1,51 @@
+- infos = Information about the xfconf plugin is in keys below
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/xfconf
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained libc configurable experimental unfinished concept limited memleak
+- infos/metadata =
+- infos/description = storage plugin for xfconf
+
+## Introduction
+
+This is a storage plugin to mount the xfconf configuration settings.
+
+## Dependencies
+
+xfconf from the XFCE project
```

Comment:
Please tell which versions you tested, how the package might be called etc.

According to your changes, dbus also seem to be important here.

Reply:
I have extended this section in the commit 0f5f8544c42fd3cbc8946a18b1834d6725973c80